osrf_testing_tools_cpp/src/memory_tools/CMakeLists.txt

```diff
@@ -51,12 +51,27 @@ endif()
 # TODO(wjwwood): If on aarch64, so not set the preload environment variables, until fixed.
 # see: https://github.com/osrf/osrf_testing_tools_cpp/issues/3
 if(NOT CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64")
+  # Option to add ASan lib path prefix to enable ASan to work as expected without interference from memory tools
+  option(OPTIONAL_ASAN_PRELOAD_ENV_VAR_OVERRIDE "Adds ASan lib path as a prefix to preload environment variable" OFF)
```

Comment:
It took a while for this to sink in but prefixing this with `OPTIONAL_` is a bit redundant (it's already an option, after all) without wanting to bikeshed further I'd suggest just dropping the `OPTIONAL_` prefix from the option name.

Reply:
Makes sense, removed the OPTIONAL_ prefix.